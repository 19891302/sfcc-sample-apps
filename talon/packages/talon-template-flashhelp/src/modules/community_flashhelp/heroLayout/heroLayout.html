<template>
    <slot name="header"></slot>
    <slot name="hero"></slot>
    <slot></slot>
    <slot name="footer"></slot>
</template>