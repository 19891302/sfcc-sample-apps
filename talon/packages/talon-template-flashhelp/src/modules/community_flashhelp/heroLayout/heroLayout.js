import { LightningElement } from 'lwc';

export default class HeroLayout extends LightningElement {}