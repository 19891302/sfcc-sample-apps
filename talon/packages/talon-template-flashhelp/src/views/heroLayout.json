{
    "name": "heroLayout",
    "label": "Hero Layout",
    "component": {
        "name": "community_flashhelp/heroLayout",
        "regions": [
            {
                "name": "header",
                "label": "Header",
                "components": [
                    {
                        "name": "community_flashhelp/header"
                    }
                ]
            },
            {
                "name": "hero",
                "label": "Hero Jumbotron",
                "components": [
                    {
                        "name": "community_flashhelp/hero"
                    }
                ]
            },
            {
                "name": "footer",
                "label": "Footer",
                "components": [
                    {
                        "name": "community_flashhelp/footer"
                    }
                ]
            }
        ]
    }
}