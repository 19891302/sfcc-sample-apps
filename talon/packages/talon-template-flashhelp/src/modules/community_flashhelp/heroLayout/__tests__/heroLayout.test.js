import { createElement } from 'lwc';
import heroLayout from 'community_flashhelp/heroLayout';

describe('community_flashhelp/heroLayout', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-hero-layout', { is: heroLayout });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});