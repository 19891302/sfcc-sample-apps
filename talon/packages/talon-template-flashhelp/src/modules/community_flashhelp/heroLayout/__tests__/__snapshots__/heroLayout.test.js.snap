// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/heroLayout renders correctly 1`] = `
<community_flashhelp-hero-layout>
  #shadow-root(open)
    <slot
      name="header"
    />
    <slot
      name="hero"
    />
    <slot />
    <slot
      name="footer"
    />
</community_flashhelp-hero-layout>
`;
